runtime-transaction/src/transaction_with_meta.rs

```diff
@@ -0,0 +1,12 @@
+use {
+    crate::transaction_meta::StaticMeta,
+    core::borrow::Borrow,
+    solana_sdk::transaction::{SanitizedTransaction, VersionedTransaction},
+    solana_svm_transaction::svm_transaction::SVMTransaction,
+};
+
+pub trait TransactionWithMeta: StaticMeta + SVMTransaction {
```

Comment:
CI still running, but I'll check w/ you now. @tao-stones @buffalojoec is this kind of what you had in mind?

Reply:
Eventually these trait fns go away and `TransactionWithMeta` will be a glorified alias

Reply:
Yessir, that's basically what I was thinking. Looks good to me, and keeps things clean at the SVM level. As long as you don't mind carrying this alias around for a bit :)